lib/cli/commands/listorders.ts

```diff
@@ -13,16 +13,16 @@ type FormattedTradingPairOrders = {
 const COLUMNS = [19, 19, 12, 19, 19, 12];
 const COLUMNS_IN_ORDER_SIDE = COLUMNS.length / 2;
 const HEADER = [
-  { content: colors.green('Buy'), colSpan: 3 },
-  { content: colors.red('Sell'), colSpan: 3 },
+  { content: colors.blue('Buy'), colSpan: 3 },
+  { content: colors.blue('Sell'), colSpan: 3 },
 ];
 const SECONDARY_HEADER = [
-  colors.green('Quantity'),
-  colors.green('Price'),
-  colors.green('Own Order'),
-  colors.red('Quantity'),
-  colors.red('Price'),
-  colors.red('Own Order'),
+  colors.blue('Quantity'),
```

Comment:
Let's keep the green and red colors here to differentiate between buy and sell.

Reply:
You resolved this convo but didn't change the colors back to green/red for the buys and sells, can you make sure this is changed?

Reply:
These lines are still being changed when it should be left as is. Please don't resolve this comment without changing it or making a comment of your own so I understand what's going on.

Reply:
`{ content: colors.green('Buy'), colSpan: 3 },
  { content: colors.red('Sell'), colSpan: 3 },`

Buy and sell are red and green.

Reply:
These columns pertain to the buy and sell sides. But these changes have been reversed so I'm resolving this.